x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/querydsl/query/SingleValueQuery.java

```diff
@@ -125,24 +131,28 @@ public static class Builder extends AbstractQueryBuilder<Builder> {
         private final QueryBuilder next;
         private final String field;
         private final Stats stats;
+        private final Source source;
 
-        Builder(QueryBuilder next, String field, Stats stats) {
+        Builder(QueryBuilder next, String field, Stats stats, Source source) {
             this.next = next;
             this.field = field;
             this.stats = stats;
+            this.source = source;
         }
 
         Builder(StreamInput in) throws IOException {
             super(in);
             this.next = in.readNamedWriteable(QueryBuilder.class);
             this.field = in.readString();
             this.stats = new Stats();
+            this.source = readSource(in);
```

Comment:
Do we need to bump the transport version for this change?

Reply:
Actually, where do we serialise this?

Reply:
> Do we need to bump the transport version for this change?

Good point, we do. I've added a new transport version.

> Actually, where do we serialise this?

The `doWriteTo()` method below updates the serialisation based on the transport version.